tests/filecheck/dialects/qubit.mlir

```diff
@@ -1,10 +1,22 @@
 // RUN: QUOPT_ROUNDTRIP
 // RUN: QUOPT_GENERIC_ROUNDTRIP
```

Comment:
The name of this file was an artefact of renaming the dialect. Could the contents of this file be moved to the new file you created `qu.mlir`.